highlight.io/components/Competitors/sentry.ts

```diff
@@ -0,0 +1,157 @@
+import sentrysmall from '../../public/images/sentry-small.png'
+import sentryfull from '../../public/images/sentry.png'
+import { Competitor } from './competitors'
+import {
+	HighlightIsOpenAndTransparent,
+	HighlightShipsNewFeatures,
+} from './shared-snippets'
+
+export const SentrySpec: Competitor = {
+	name: 'Sentry',
+	header: 'A Truly Open Source Sentry Alternative.',
+	subheader:
+		'Session replay of your frontend, fullstack error monitoring, and powerful logging.',
+	logoDesktop: sentryfull,
+	logoMobile: sentrysmall,
+	sections: [
+		{
+			title: 'General',
+			rows: [
+				{
+					feature: 'Unlimited Team Members',
+					highlight: 1,
+					competitor: 1,
+				},
+				{
+					feature: 'Support Ticket Integrations',
+					tooltip: 'Create tickets with clickup, linear, or jira.',
```

Comment:
We don't support jira (yet).

`Create tickets with Linear, GitHub, Height, or ClickUp`